matrix_benchmarking/store/__init__.py

```diff
@@ -140,4 +142,14 @@ def register_custom_rewrite_settings(fn):
     global custom_rewrite_settings
     custom_rewrite_settings = fn
 
+
+def register_custom_schema(model):
+    global custom_schema
+    custom_schema = model
+
+def _get_custom_schema():
```

Comment:
```suggestion
def get_custom_schema():
```
it's not a package-internal method as it's used in `matrix_benchmarking/export_schema.py`